src/modules/ekf2/EKF2.cpp

```diff
@@ -1368,9 +1377,6 @@ void EKF2::PublishStatusFlags(const hrt_abstime &timestamp)
 		status_flags.reject_ver_vel                  = _ekf.innov_check_fail_status_flags().reject_ver_vel;
 		status_flags.reject_hor_pos                  = _ekf.innov_check_fail_status_flags().reject_hor_pos;
 		status_flags.reject_ver_pos                  = _ekf.innov_check_fail_status_flags().reject_ver_pos;
-		status_flags.reject_mag_x                    = _ekf.innov_check_fail_status_flags().reject_mag_x;
```

Comment:
Then also remove them from the status flags and the uorb msg?

Reply:
I wasn't going to change any of the existing flags until we discussed a path forward for Tools/ecl_ekf (and Auterion/ecl_ekf_analysis).

Reply:
I've updated estimator_status_flags for now.